fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchAllTest.java

```diff
@@ -154,7 +175,7 @@ public void testSearchAllUsingSecurity() throws Exception {
     public void testSearchAllUsingProfile() throws Exception {
         FHIRParameters parameters = new FHIRParameters();
         parameters.searchParam("_profile", "http://ibm.com/fhir/profile/Profile");
-        FHIRResponse response = client.searchAll(parameters);
+        FHIRResponse response = client.searchAllGet(parameters);
```

Comment:
I'm looking at the signature some more.  is there a better option?  where we can have something like this? 

client.searchAll(parameters) ... goes to post. 
client.searchAll(parameters, boolean) 
true goes to post
false goes to get

Reply:
make sense, will change

Reply:
done